astrocut/cutout_processing.py

```diff
@@ -425,8 +425,12 @@ def center_on_path(path, size, cutout_fles, target=None, img_wcs=None,
         target = "path" if not target else target
         target_pixel_file = (f"{target}_{primary_header['TSTART']}-{primary_header['TSTop']}_"
                              f"{size[0]}-x-{size[1]}_astrocut.fits")
+    
+    # Replace any slashes/spaces with % for filename conventions
+    target_pixel_file = target_pixel_file.replace('/', '%').replace(' ', '_')
```

Comment:
I think we might have more cross-platform compatibility and avoid some issues if we use `-` instead of `%` here.